### PR TITLE
Improve clone performance and remove dependency by composer

### PR DESCRIPTION
**Pull request by mhz-tamb**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -4,45 +4,44 @@ help: ## Help
...
+	# copy example asterisk config for running 
 	cp -R docker/asterisk/conf/* rbt/install/asterisk/
-# cope default enviroments
+	# cope default enviroments
 	cp .example.env .env
 
 rbt_containers_up: ## Start RBT containers
 	docker compose up -d
-rbt_containers_down: ##  Stop RBT containers
+	rbt_containers_down: ##  Stop RBT containers
```
Comment (kulakoff) [inline Makefile]: remove space character

File: Makefile [hunk 2]
```diff
@@ -4,45 +4,44 @@ help: ## Help
...
 	docker compose down	
-rbt_syslog_up: ## Start RBT Syslog service
+	rbt_syslog_up: ## Start RBT Syslog service
```
Comment (kulakoff) [inline Makefile]: remove space character

File: Makefile [hunk 3]
```diff
@@ -4,45 +4,44 @@ help: ## Help
...
 	docker-compose -f docker-compose-syslogs.yml up -d
-rbt_syslog_down: ## Stop RBT Syslog service
+	rbt_syslog_down: ## Stop RBT Syslog service
```
Comment (kulakoff) [inline Makefile]: remove space character

**Review by kulakoff — COMMENTED** [review]
invalid makefile syntax, please fix :+1:

**Comment by mhz-tamb** [comment]
Done

**Review by kulakoff — COMMENTED** [review]
all ok :)

**Pull request merged**